cbits-extra/hs-zlib.h

```diff
@@ -0,0 +1,8 @@
+#ifndef HS_ZLIB_EXTRAS
+#define HS_ZLIB_EXTRAS
```

Comment:
What is this for?

Reply:
It is an [include guard](https://en.wikipedia.org/wiki/Include_guard). Alternatively, this could be replaced with [`#pragma once`](https://en.wikipedia.org/wiki/Pragma_once) (with very(?) slightly reduced portability).